coalib/__init__.py

```diff
@@ -7,10 +7,46 @@
 
 import sys
 from os.path import join, dirname
+from subprocess import check_output, CalledProcessError
 
 
 VERSION_FILE = join(dirname(__file__), 'VERSION')
 
+COMMIT_FILE = join(dirname(__file__), 'COMMIT')
+
+
+def get_current_commit():
+    '''
+    :return:    string containing the current commit sha if coala is
+                initialized as a git repo otherwise an empty string
+    '''
+    try:
+        commit = check_output(['git', 'rev-parse', 'HEAD'])
```

Comment:
Use `sarge` instead of using `check_output`, as `sarge` provides a sane and safe cross platform interface.